zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -109,13 +109,10 @@ pub async fn run() -> Result<()> {
         "got transactions to send",
     );
 
-    // TODO: change debug_skip_parameter_preload to true if we do the mempool test in the wallet gRPC test
-    let (mut zebrad, zebra_rpc_address) = spawn_zebrad_for_rpc_without_initial_peers(
-        Network::Mainnet,
-        zebrad_state_path,
-        test_type,
-        false,
-    )?;
+    // Start zebrad with no peers, we want to send transactions without blocks coming in. If `wallet_grpc_test`
+    // runs before this test (as it does in `lightwalletd_test_suite`), then we are the most up to date with tip we can.
```

Comment:
In CI, these tests run in parallel, so we'll need to change the logic in `wait_for_zebrad_and_lightwalletd_tip` to check `lightwalletd` is at the tip, and add it to `send_transaction_test::run()`.